web/src/main/java/org/akaza/openclinica/controller/StudyParticipantController.java

```diff
@@ -105,6 +106,7 @@
 		@RequestMapping(value = "/{studyOID}/participants/bulk", method = RequestMethod.POST,consumes = {"multipart/form-data"})
 		public ResponseEntity<Object> createNewStudyParticipantAtStudyLevel(HttpServletRequest request, 
 				@RequestParam("file") MultipartFile file,
+				//@RequestPart("json") Optional<JsonPojo> map,
```

Comment:
Please put a comment above this saying we will implement this class when we decide to pass additional parameters